Task1/src/main/java/org/example/task1/reader/XmlReaderService.java

```diff
@@ -0,0 +1,163 @@
+package org.example.task1.reader;
+
+
+import org.example.task1.model.XmlEmployee;
+import org.example.task1.model.XmlEmployees;
+import org.example.task1.model.XmlProject;
+import org.example.task1.model.XmlProjects;
+import org.springframework.stereotype.Service;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.SAXException;
+
+import javax.xml.XMLConstants;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import java.io.File;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class XmlReaderService {
+
+    private static final Logger logger = LoggerFactory.getLogger(XmlReaderService.class);
+
+    private final JAXBContext employeesContext;
+    private final JAXBContext projectsContext;
+
+    public XmlReaderService() throws JAXBException {
+        this.employeesContext = JAXBContext.newInstance(XmlEmployees.class);
+        this.projectsContext = JAXBContext.newInstance(XmlProjects.class);
+    }
+
+
+    public Optional<List<XmlEmployee>> readEmployees(String filePath) {
```

Comment:
We should have XmlReader, and this Reader should have only one method read. Pls rewrite this